build-system/tasks/helpers.js

```diff
@@ -218,31 +218,29 @@ function compile(watch, shouldMinify) {
     );
   }
 
-  if (!watch || argv.with_inabox) {
-    if (!argv.single_pass) {
-      promises.push(
-        // Entry point for inabox runtime.
-        compileJs('./src/inabox/', 'amp-inabox.js', './dist', {
-          toName: 'amp-inabox.js',
-          minifiedName: 'amp4ads-v0.js',
-          includePolyfills: true,
-          extraGlobs: ['src/inabox/*.js', '3p/iframe-messaging-client.js'],
-          watch,
-          minify: shouldMinify,
-        })
-      );
-    }
+  if (!argv.single_pass) {
```

Comment:
Does inabox not compile in single pass mode? If it does, you can remove this `if` block as well.

/cc @erwinmombay

Reply:
will leave that to @erwinmombay

Reply:
I just realized these targets are built separately even during single pass compilation, so it's safe to remove `if(!argv.single_pass)`.

You can test this by removing the `if` guard and running:
```
gulp dist --single_pass
```
If it works, let's roll all these into the main `promises` array. :)

Reply:
works